README.md

```diff
@@ -7,3 +7,7 @@ test base classes and documentation shared by many XBlocks.
 To test the utilities, run:
 
     nosetests --with-coverage --cover-package xblockutils --cover-html
```

Comment:
No, it wasn't working for me at all. I think because it doesn't have the Django environment configured properly. That's why I added the alternate command below, which works fine. I left the old one in because I wasn't sure what I was missing, and the alternate I've suggested doesn't generate a coverage report.